app/settings.py

```diff
@@ -146,6 +146,8 @@ def utcoffset_or_fail(date_value, key):
 
 SDS_API_BASE_URL = os.getenv("SDS_API_BASE_URL")
 
+CIR_API_BASE_URL = os.getenv("CIR_API_BASE_URL")
```

Comment:
This needs adding to the README https://github.com/ONSdigital/eq-questionnaire-runner?tab=readme-ov-file#environment-variables
Looks like `SDS_API_BASE_URL` was previously missed, worth adding that also.

Reply:
Good spot, added both